docs/api/data-models/service-orchestration-flag.md

```diff
@@ -0,0 +1,10 @@
+# ServiceOrchestrationFlag
+
+Specific String:Boolean pair to control the orchestration process. Possible values:
+
+- `MATCHMAKING`: If **true** orchestration process includes a matchmaking process in order to return only one matching service instance if any. If **false**, orchestration process returns all the matching service instance if any.
+- `ONLY_PREFERRED`: If **true**, orchestration process considers only those matching service instances that are provided by a preferred provider if any. If **false**, but preferred providers are specified and have matching service instances, than orchestration process considers only those service instances that are provided by a preferred provider. Otherwise, non-preferred providers are considered.
+- `ONLY_EXCLUSIVE`: If **true**, orchestration process considers only those matching service instances that are allows exclusivity. It automatically results `MATCHMAKING:true` as well. If **false**, but exclusivity duration is specified and there are matching services with exclusivity allowed, than orchestration process considers only those service instances that allows exclusivity. Otherwise, service instances without exclusivity are considered.
+- `ALLOW_INTERCLOUD`: If **true**, orchestration process considers matching service instances from neighbor clouds when there are no local hits. Orchestrating from neighbor clouds automatically results `MATCHMAKING:true`. If **false**, orchestration process doesn't consider matching service instances from neighbor clouds when there are no local hits.
+- `ONLY_INTERCLOUD`: If **true**, orchestration process considers matching service instances only from the neighbor clouds. It automatically results `MATCHMAKING:true` as well. If **false**, orchestration process considers matching service instances from the local could in first hand and only considers matching service instances from the neighbor clouds when `ALLOW_INTERCLOUD:true`.
+- `ALLOW_TRANSLATION`: If **true**, orchestration process considers matching, but non-native service instances when there are no native hits. It automatically results `MATCHMAKING:true` as well. If **false**, orchestration process considers only native matching service instances.
```

Comment:
I understand what you mean when you use the words native and non-native, but normal users possible will not. Somehow you should include a definition.